docs/sbpy/units.rst

```diff
@@ -0,0 +1,61 @@
+Photometry Module (`sbpy.units`)
+=====================================
+
+Introduction
+------------
+
+`~sbpy.units` provides common planetary astronomy units and Vega-based
+magnitude conversions.  ``sbpy`` units may be added to the top-level
+``astropy.units`` namespace via:
+
+  >>> import sbpy.units
+  >>> sbpy.units.enable()    # doctest: +SKIP
+
+This will make them searchable via strings, such as
+``u.Unit('mag(VEGA)')``.
+
```

Comment:
Just a heads-up, later I will add some reflectance units such as I/F, RADF, REFF, etc to this submodule, and update the description here.

Reply:
I also have a ``hundred_nm`` unit to add for spectral gradients.